tests/sim/test_splat.py

```diff
@@ -4,6 +4,9 @@
 
 N = 1000
 M = 10000
+batch = (M // 2) * ["A"] + (M // 2) * ["B"]
+group = (M // 4) * ["A"] + (M // 4) * ["B"]
+group = group + group
```

Comment:
why do we do group = group+group ? and if it is needed why don't we have to do batch = batch+batch

Reply:
I just wanted to avoid that groups and batches overlap.
First batch is the 1st half of the samples, second batch is the 2nd half of the samples.
First groups is the 1st and 3rd quarters of the samples, second group is the 2nd and 4th quarters of the samples.

Reply:
thank you for the explanation